app/crawler/mbc_hybrid.py

```diff
@@ -0,0 +1,25 @@
+# app/crawler/mbc_hybrid.py
+
+from app.crawler.base import BaseNewsCrawler, Article
+from app.crawler.mbc_api import MbcNewsApiCrawler
+from app.crawler.mbc_html import MbcHtmlCrawler
+
+class HybridMbcCrawler(BaseNewsCrawler):
+    """
+    MBC 뉴스 크롤러 (API 우선, 실패 시 HTML fallback)
+    """
+
+    def __init__(self):
+        self.api_crawler = MbcNewsApiCrawler()
+        self.html_crawler = MbcHtmlCrawler()
+
+    async def fetch_articles(self, keyword: str) -> list[Article]:
+        try:
+            print("[Hybrid] Trying API crawler...")
```

Comment:
print 대신 logging을 사용해볼까요?

app.crawler.utils 에 logger를 하나 만들고 공통으로 사용하면 좋을 것 같습니다.

```suggestion
import logging
logging.basicConfig(
    level=logging.INFO,
    format="%(asctime)s - %(levelname)s - %(message)s",
    datefmt="%Y-%m-%d %H:%M:%S",
    handlers=[
        # 파일에 기록하려면 추가
        # logging.FileHandler("news-alert-system.log"),
        logging.StreamHandler(),
    ],
)
```

Reply:
common/utils/logger.py 를 만들어  프로젝트 전반에 사용할 logger 를 만들었습니다. 
덕분에 크롤러 함수 전반에 logger를 사용하는 방법도 알게되었습니다. 
그리고 logger 코드 덕분에 주석을 어느정도 대체할 수 있어, 자세한 주석을 로거가 대체할 수 도 있겠다라는 생각을 하게 되었습니다.